src/ImageSharp/Processing/Processors/Transforms/RotateProcessor.cs

```diff
@@ -87,6 +88,30 @@ protected override void BeforeApply(ImageFrame<TPixel> source, Rectangle sourceR
             }
         }
 
+        /// <inheritdoc/>
+        protected override void AfterImageApply(Image<TPixel> source, Rectangle sourceRectangle)
+        {
+            ExifProfile profile = source.MetaData.ExifProfile;
+            if (profile == null)
+            {
+                return;
+            }
+
+            if (MathF.Abs(this.Angle) < Constants.Epsilon)
+            {
+                // No need to do anything so return.
+                return;
+            }
+
+            profile.RemoveValue(ExifTag.Orientation);
+
+            if (this.Expand && profile.GetValue(ExifTag.PixelXDimension) != null)
+            {
+                profile.SetValue(ExifTag.PixelXDimension, source.Width);
+                profile.SetValue(ExifTag.PixelYDimension, source.Height);
```

Comment:
It is possible that we add the `ExifTag.PixelYDimension` value. And I wonder if we should do this operation. Otherwise we need to this in a lot of spots. I personally think we shouldn't do this.

Reply:
I made the assumption that if one exists, both will. Why wouldn't you do this? I had users asking for EXIF parity with ImageProcessor.

Reply:
Doesn't this mean that you would also need to add something to the resize processor? Maybe we could do this logic inside `Sync` method of the ExifProfile if you really think we need this?

Reply:
We would yeah, crop also. I'd love to have it centralized but how would it be aware of the required changes?